index.html

```diff
@@ -120,70 +120,89 @@ <h1 class="Projeect-Title">Multi-Post
+                                <li><a href="#"><i class="bi bi-github"></i></a></li>
+                                <li><a href="#"><i class="bi bi-slack"></i></a></li>
+                            </ul>
+                        </section>
+                        <button class='btn' type="button">Get my resume</button>
+                    </article>
+                    <ul id="my-info">
+                        <li class="list resume-languages">
+                          <div class="lang-container">
+                            <p>
+                              Languages
+                             </p>
+                             <img src="assets/icon-down.png" alt="">
+                          </div>
+                          <ul class="lang-list">
+                            <li class="list-item">
+                                <img src="assets/javascript.png" alt="">
+                                JavaScript</li>
+                            <li class="list-item">
+                                <img src="assets/css.png" alt="">
+                                HTML</li>
+                            <li class="list-item"><img src="assets/html.png" alt="">CSS</li>
+                          </ul>
+                        </li>
+                        <li class="resume-list frameworks">
+                          <div class="lang-container">
+                            <p>
+                              Frameworks
+                            </p>
+                            <img src="assets/icon-right.png" alt=""></div>
+                        </li>
+                        <li class="resume-list skills">
+                          <div class="lang-container">
+                            <p>
+                              Skills
+                            </p>
+                            <img src="assets/icon-right.png" alt="">
+                          </div>
+                        </li>
+                      </ul>
+                </section>
+                <section class="contacts">
+                 <div>
+                  <form class="forms" action="https://formspree.io/f/xzbooavl" method="POST">
+                      <h2 class="contactme-head">
+                        Contact me
+                      </h2>
+                      <p class="contactme-body">
+                        If you have an application
+                        you are interested in developing,
+                          a feature that you need built
+                          or a project that needs coding.
+                          I’d love to help with it </p>
+                        <input class="form-name" type="text" maxlength="30" placeholder="Full Name" required>
+                        <input class="form-email" type="email" name="email" placeholder="Email">
+                        <textarea class="form-message" name="message" maxlength="500" placeholder="Write your message here"></textarea> <br>
```

Comment:
- [ ] Kindly note that all input fields are required. Add the `required` attribute to these input fields.